templates/zookeeper-dns.yaml

```diff
@@ -0,0 +1,241 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Hyku stack for zookeeper dns
+Parameters:
+  ZookeeperEnvironmentName:
+    Type: String
+    Description: Environment name from zookeeper stack
+  ZookeeperRecordName:
+    Type: String
+    Description: Environment name from zookeeper stack
+  ZookeeperHostedZoneID:
+    Type: String
+    Description: Environment name from zookeeper stack
+  SubnetID:
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: Private subnets for zk stack
+  SecurityGroups:
+    Type: List<AWS::EC2::SecurityGroup::Id>
+    Description: Private subnets for zk stack
+Resources:
+  BeanstalkStack:
+    Type: Custom::BeanstalkStack
+    Properties:
+      ServiceToken: !GetAtt [BeanstalkStackOutputs, Arn]
+      EnvironmentName: !Ref ZookeeperEnvironmentName
+  BeanstalkStackOutputs:
+    Type: AWS::Lambda::Function
```

Comment:
Would be helpful to have a comment here describing the purpose of this Lambda function